const test = require('ava');

test('this is a failing test', t => t.fail());
